src/main/java/wolox/training/models/User.java

```diff
@@ -0,0 +1,83 @@
+package wolox.training.models;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+import java.util.Collection;
+
+
+@Entity
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Column(nullable = false)
+    private String userName;
+
+    @Column(nullable = false)
+    private String name;
+
+    @Column(nullable = false)
+    private LocalDate birthDate;
+
+    @OneToMany
```

Comment:
The relationship should be a ManyToMany one. 
Here you can see it better. https://en.wikipedia.org/wiki/Many-to-many_(data_model)